std/experimental/allocator/mallocator.d

```diff
@@ -210,7 +225,8 @@ struct AlignedMallocator
     $(WEB msdn.microsoft.com/en-us/library/8z34s9c6(v=vs.80).aspx,
     $(D __aligned_malloc)) on Windows.
     */
-    version(Posix) @trusted @nogc
+    version(Posix)
+    @trusted @nogc nothrow
```

Comment:
This isn't memory safe when asserts are disabled. `posix_memalign` fails with `EINVAL` when given a poor alignment argument, which isn't handled.